frontend/pkg/database/database.go

```diff
@@ -43,13 +56,13 @@ func NewDatabaseConfig(dbName, dbURL string) *DBConfig {
 }
 
 // NewDatabaseClient instantiates a Cosmos DatabaseClient targeting Frontends async DB
-func NewDatabaseClient(config *DBConfig) (*DBClient, error) {
+func NewDatabaseClient(config *DBConfig) (*CosmosDBClient, error) {
```

Comment:
Same as with `NewCache`: Suggest returning `DBClient` here, since it doesn't look like you can do anything with a `CosmosDBClient` beyond what the `DBClient` interface provides.